manifests/init.pp

```diff
@@ -315,8 +340,15 @@
   $template_snmptrapd           = $snmp::params::template_snmptrapd,
   $template_snmptrapd_sysconfig = $snmp::params::template_snmptrapd_sysconfig,
   $openmanage_enable            = $snmp::params::openmanage_enable,
+  $master                       = $snmp::params::master,
+  $agentx_perms                 = $snmp::params::agentx_perms,
+  $agentx_ping_interval         = $snmp::params::agentx_ping_interval,
+  $agentx_socket                = $snmp::params::agentx_socket,
+  $agentx_timeout               = $snmp::params::agentx_timeout,
+  $agentx_retries               = $snmp::params::agentx_retries,
 ) inherits snmp::params {
   # Validate our booleans
+  validate_bool($master)
```

Comment:
* Readme fixed
* Added validation / new defaults for two of the AgentX options that had defaults listed in the man page.  The other options have no defaults (i.e. undef)
* Fixed duplicate variable from bad rebase